src/key-binding.coffee

```diff
@@ -77,7 +77,7 @@ class KeyBinding
             return false
 
       if isPartialMatch
-        bindingRemainderContainsOnlyKeyups = false unless bindingKeystroke.startsWith('^')
+        bindingRemainderContainsOnlyKeyups = false unless bindingKeystroke.startsWith('^') and bindingKeystroke.length > 1
```

Comment:
There is a specific `isKeyup` function. What do you think about using that instead?

https://github.com/atom/atom-keymap/blob/3b17ede7f10e5a9c5097b495a03e88e2c7bacfa3/src/helpers.coffee#L264